agreement/voteTracker.go

```diff
@@ -116,8 +117,21 @@ func (tracker *voteTracker) handle(r routerHandle, p player, e0 event) event {
 		}
 
 		sender := e.Vote.R.Sender
-		_, equivocator := tracker.Equivocators[sender]
+		eqVote, equivocator := tracker.Equivocators[sender]
 		if equivocator {
+			equivocationDetails := telemetryspec.EquivocatedVoteEventDetails{
+				VoterAddress:          sender.String(),
+				ProposalHash:          e.Vote.R.Proposal.BlockDigest.String(),
+				Round:                 uint64(e.Vote.R.Round),
+				Period:                uint64(e.Vote.R.Period),
+				Step:                  uint64(e.Vote.R.Step),
+				StepThreshold:         e.Vote.R.Step.threshold(proto),
```

Comment:
Extraneous but I guess it's fine

Reply:
If the threshold is actually extraneous @tsachiherman then we don't need to pipe in the proto through the voteFilterRequest